### ISSUE TITLE
plotQualityProfile scaling issue

### ISSUE THREAD

**Issue by termithorbor** [benjjneb/dada2#1263]
Hi,

Any idea why the read line is not starting at 100 %?

![image](https://user-images.githubusercontent.com/66436023/106756946-1adbae00-6630-11eb-9912-d65b2c3c1523.png)

Is this a bug?

Thanks in advance.

**Comment by benjjneb** [comment]
It looks like a bug, but what version of the package are you using? A bug that affected where the read line started was fixed in the last release or two.

**Comment by termithorbor** [comment]
I am using version 1.16.0 - yes probably I should update to 1.18 then.

**Comment by benjjneb** [comment]
Yes I think the bug I was thinking of was fixed after the 1.16 release: e18873e63c8fddb6e351f31647efca8e99dedbe4

**Comment by termithorbor** [comment]
Thank you very much :)
